graphql/mappers/my-dashboard.js

```diff
@@ -15,6 +15,7 @@ export async function getMyDashboardContent() {
           return {
             id: fragment.scId,
             title: fragment.scTitleEn,
+            dropdownText: fragment.schTasks[0].scLinkTextEn,
```

Comment:
Would it be possible to get this by scid?

Reply:
Approving since IDs won't match per card

Reply:
Discussed, will need to change the IDs in AEM